src/collections/projects/service-mesh-interface/index.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: "Service Mesh Interface Conformance"
+subtitle: "Layer5 is a prominent maintainer and supporter of SMI, advancing its specifications and delivering conformance tooling through Meshery."
+published: true
+---
+
+import { ProjectWrapper } from "../Project.style.js"
+import { Container, Row, Col } from "../../../reusecore/Layout"
+import smi1 from "./checklist.svg"
+import Button from "../../../reusecore/Button"
+
+<ProjectWrapper>
+  <Row className = "smi-item">
+    <div className="center smi-top-left">
+    <img
+      src= {smi1}
+      width="70%" 
+    />
+    </div>
+  
+    <Button className = "smi-button">
+      <a
+        href="/"><h4 className = "btn-title">Run SMI Conformance</h4></a>
```

Comment:
Here the `Button` component can be used. P.S. Ignore the purple color you get used you use it. It will be fixed later

Reply:
Sure, I will change it.